docs/api/users.md

```diff
@@ -104,3 +104,36 @@ Content-Type: application/json
   }
 ]
 ```
+
+
+## Get current user
+
+### URL
+
+```http
+GET /user HTTP/1.1
+```
+
+### Request
+
+```http
+GET /user HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+  "id": "bgfg83bjdg62j",
+  "firstName": "Juriaan",
+  "infix": null,
+  "lastName": "Winkens",
+  "roles": ["STUDENT"],
+  "created": "2017-01-03T12:16:55Z",
+  "updated": "2017-01-03T12:16:55Z"
+}
```

Comment:
In #22 the `organisationId` is removed, is there a reason to add it here? It feels like it would make sense, but I'd rather have a logical reason instead. @Lehren any ideas?